src/components/header/header.js

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import { Icon } from 'react-fa';
+import './header.scss';
+import Logo from '../../components/logo/logo';
+import SlackLogo from '../../images/Slack-Monochrome-White.svg';
```

Comment:
@Peter-holyoke 
Let's keep all file names consistent, e.g. _slack-monochrome-white.svg_  (all lowercase)

Reply:
Will do. Also, I think it makes sense to use all SVGs for the social Icons. I'll discuss this with Elana, since it looks like it will require a change to one of the icons.